sound/soc/codecs/rt715-sdca-sdw.c

```diff
@@ -70,6 +70,7 @@ static bool rt715_sdca_mbq_readable_register(struct device *dev, unsigned int re
 	case 0x2000036:
 	case 0x2000037:
 	case 0x2000039:
+	case 0x2000044:
```

Comment:
@jack-cy-yu Could you describe why this patch can fix the clock stop prepare timeout issue in the commit message? Also, I don't think "Device should be powered down and reboot for this fix to take effect." is needed in the commit message.

Reply:
@bardliao I've updated the commit message, please check it. Thanks.

Reply:
I would suggest the following commit message 

````
The Clock_Stop_Not_Finished bit is currently not cleared during the clock stop sequence, which results in the clock stop sequence timing out after 300ms. This is root-caused to a missing register configuration impacting the transition between the bus clock and an internal clock.

This problem was exposed by cleanups in the SoundWire bus code, where previously the ClockStopPrepare was mistakenly skipped when using ClockStopMode0.

Tested-by: Pierre-Louis Bossart <pierre-louis.bossart@linux.intel.com>
BugLink: https://github.com/thesofproject/linux/issues/2853
Fixes: 20d17057f0a8 ("ASoC: rt715-sdca: Add RT715 sdca vendor-specific driver")
````